src/lib.rs

```diff
@@ -312,53 +330,6 @@ impl ShortRange {
     }
 }
 
-/// A custom variant of `Cow<'a, str>` that
```

Comment:
Are you sure we don't need this anymore?

Reply:
`ExpandedNameOwned` is gone and was the only user. Applying it elsewhere does not have clear benefits (I tried).

Reply:
Oh, I see. That was a nice trick.

Reply:
Can't we use it for `AttributeData.value`?

Reply:
We could, but both `Cow<'input, str>` and `CowStr<'input>` are 24 bytes on their own, so the struct itself would not get smaller. (Only `Option<CowStr<'input>>` is smaller than `Option<Cow<'input, str>>`.)

The main difference would be that we should any excess capacity of the owned string, but due to allocator size classes this does not actually save memory most of the time. So I think this alone is not worth the trouble of having a bespoke `Cow` variant.

Reply:
Go it.